### ISSUE TITLE
css themes for IPython

### ISSUE THREAD

**Issue by DaveBackus** [DaveBackus/Data_Bootcamp#8]
Low priority, but Brandon Rhodes uses one to make the display of df's sharper.  I'm curious how this works.

**Comment by sebecketthile** [comment]
Hm...I've used CSS with pandas before, but only after converting the table
to HTML.  The examples I'm seeing online for css with iPython seem to
affect the whole page, not just the dfs. Can you send examples of what you
have in mind?

On Tue, Nov 10, 2015 at 8:31 AM, Dave Backus <notifications@github.com>
wrote:

> Low priority, but Brandon Rhodes uses one to make the display of df's
> sharper. I'm curious how this works.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DaveBackus/Data_Bootcamp/issues/8>.
>



-- 

*Sarah Beckett-Hile*

*MBA Candidate, Class of 2016*

NYU Leonard N. Stern School of Business

916.768.8492 <+916.768.8492> | sbh355@stern.nyu.edu

   - www.linkedin.com/in/sarahbecketthile/

**Comment by cc7768** [comment]
I saw some cool style stuff that they just released or are planning to release soon. I'll find the slides and post here a little later.

**Comment by spencerlyon2** [comment]
https://github.com/jreback/PyDataNYC2015/blob/master/whats-new-in-pandas/style.ipynb

**Comment by spencerlyon2** [comment]
This link works:

**Comment by sebecketthile** [comment]
Ah this is awesome!

On Tue, Nov 10, 2015 at 9:36 AM, Spencer Lyon <notifications@github.com>
wrote:

> This link works:
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DaveBackus/Data_Bootcamp/issues/8#issuecomment-155436693>
> .
>



-- 

*Sarah Beckett-Hile*

*MBA Candidate, Class of 2016*

NYU Leonard N. Stern School of Business

916.768.8492 <+916.768.8492> | sbh355@stern.nyu.edu

   - www.linkedin.com/in/sarahbecketthile/
